css-positioning/absolute.css

```diff
@@ -0,0 +1,45 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+  position: relative;
+
+  /* your code here */
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+
+
+  /* your code here */
+  position: absolute;
+  left: 230px;
```

Comment:
Please revise this to use smaller measurements from `right` as we discussed in Q&A

Reply:
![Screen Shot 2021-01-22 at 3 16 33 PM](https://user-images.githubusercontent.com/74885918/105558980-d9a8dd00-5cc4-11eb-8e77-3ed22556bd2b.png)